tests/bluetooth/bsim_bt/bsim_test_iso/src/main.c

```diff
@@ -85,13 +96,15 @@ static void test_iso_main(void)
 		FAIL("Failed to enable periodic advertising (err %d)\n", err);
 		return;
 	}
+	printk("success.\n");
 
-	/* Start extended advertising */
+	printk("Start extended advertising...\n");
```

Comment:
Typically you don't have the `\n` when you do these `Doing x... success` printk's

Reply:
Fixed.